composer.json

```diff
@@ -52,5 +52,11 @@
     "psr-4": {
       "Lctrs\\MinkPantherDriver\\Test\\": "test/"
     }
-  }
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/Lctrs/driver-testsuite.git"
```

Comment:
This won't be loaded in any project using this library, because of https://getcomposer.org/doc/05-repositories.md#repository
>Repositories are only available to the root package and the repositories defined in your dependencies will not be loaded.

Reply:
This is just for a dev dependency so that's not a problem.

Anyway, I won't merge this as-is. This was added to load https://github.com/FriendsOfBehat/driver-testsuite/pull/2. I won't merge this until that PR and https://github.com/FriendsOfBehat/driver-testsuite/pull/1 are merged.